src/tools/miri/src/diagnostics.rs

```diff
@@ -359,12 +359,14 @@ fn report_msg<'tcx>(
     }
     // Add backtrace
     for (idx, frame_info) in stacktrace.iter().enumerate() {
-        let is_local = machine.is_local(frame_info);
         // No span for non-local frames and the first frame (which is the error site).
-        if is_local && idx > 0 {
+        if idx > 0 {
```

Comment:
Removing this `is_local` check blows up the size of Miri backtraces with information which is not useful to the user. Is there a principled reason we can't have this check anymore?

Reply:
nope ^^ I just forgot about that